cognoml/tests/unit_tests/test_figshare.py

```diff
@@ -0,0 +1,12 @@
+import os
+import re
+from cognoml import figshare
+
+def test_get_article_versions():
+    article_url = 'https://api.figshare.com/v2/articles/3487685/versions'
+    url_pattern = re.compile("https://api.figshare.com/v2/articles/([0-9]*)/versions/([0-9]*$)", re.MULTILINE)
+    version_to_url = figshare.get_article_versions(3487685)
+    assert type(version_to_url) == dict
```

Comment:
Yes. OrderedDict is a dict subclass. I figured the version_to_url doesn't necessarily need to be in sorted order.